modules/gsp-cluster/data/values.yaml

```diff
@@ -152,6 +152,7 @@ fluentd-cloudwatch:
 
 gsp-monitoring:
   prometheus-operator:
+    kubeTargetVersionOverride: "${eks_version}"
```

Comment:
This is an undocumented workaround for not using `helm install` (see, e.g., https://github.com/helm/charts/blob/4e9b398336756ad24baab6512937e75596602a30/stable/prometheus-operator/templates/prometheus/rules-1.14/alertmanager.rules.yaml#L4-L5)